src/styles/blocks/star.css

```
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
```

one center will be enough)
```suggestion
  background-position: center;
```